templates/templated_email/source/order/partials/_order_lines.mjml

```diff
@@ -36,10 +36,10 @@
       </tfoot>
       <thead class="table-header-row">
         <tr>
-          <th>Item</th>
-          <th>Quantity</th>
-          <th>Per unit</th>
-          <th>Subtotal</th>
+          <th>{% trans "Item" context "E-mail order line summary header" %}</th>
+          <th>{% trans "Quantity" context "E-mail order line summary header" %}</th>
+          <th>{% trans "Per unit" context "E-mail order line summary header" %}</th>
+          <th>{% trans "Subtotal" context "E-mail order line summary header" %}</th>
```

Comment:
I think these could be improved by stating what the noun describes rather than where it appears. Example:

```django
{% trans "Per unit" context "Unit price of a product" %}
```

I'm also not sure why the column headers here differ from the ones that appear in order history and the dashboard. Perhaps we could reuse both the names and the translation context used there (and spare translators from having to retranslate the same words over and over again)?

Reply:
I agree with you.

For the order on the frontstore, we have:
```django
{# templates/order/_ordered_items_table.html #}

{% trans "Product" context "Order details column title" %}
{% trans "Quantity" context "Order details column title" %}
{% trans "Price" context "Order details column title" %}
```

I'm thinking we could keep and copy it all inside the email template. But we should keep the `Per Unit` price. Or maybe add it as well in the frontstore and dashboard?

Also, should we use `Price` or `Subtotal`? I feel like `Subtotal` would be more descriptive.

Reply:
Hmm, what information is actually useful to the person? This is not an invoice so not all columns need to be present. Is it better to say "Item, 2 × $1.20" or "Item × 2, $2.40"? Opinions?

Storefront order details and the email should be consistent in this regard.

Reply:
I agree, I would be more for `Item × 2, $2.40`

Reply:
I agree with @patrys about using more meaningful context for translations. Unifying order columns is another problem that we should open an issue for and solve separately. @NyanKiyoshi Will you have time to update the contexts here? If you're busy with anything else I'll take care, just let me know.

Reply:
I agree, that would probably be better to do that in another PR. I will do that once I'm off of work.

Reply:
Hopefully it should be good now. Have a great night!

Reply:
Thanks!